validator/keymanager/types.go

```diff
@@ -122,3 +136,17 @@ func ParseKind(k string) (Kind, error) {
 		return 0, fmt.Errorf("%s is not an allowed keymanager", k)
 	}
 }
+
+// DisplayRemotePublicKeys prints remote public keys to stdout.
+func DisplayRemotePublicKeys(validatingPubKeys [][48]byte) {
```

Comment:
still wondering if it should be in this package but good job catching that they are used in both places

Reply:
I agree. I moved this to a new package called `remote_utils` and a file called format.go. Let me know if you prefer something else!